src/js/vre/chapter36/config/form.js

```diff
@@ -221,10 +221,20 @@ const formConfig = {
                   hideIf: (formData) => formData['view:isVeteran'] === true,
                 }
               }
-            )
+            ),
+            applicantGender: {
+              'ui:title': 'Gender',
+              'ui:widget': 'radio',
+              'ui:required': formData => formData['view:isVeteran'] === true,
+              'ui:options': {
+                labels: genderLabels,
+                hideIf: ({ 'view:isVeteran': isVeteran }) => !isVeteran
```

Comment:
Could we make this match the other hideIf function above (but negated)? Seems like it'll be easier to read if they're written the same way.

Reply:
Sounds good, changed.